### PR TITLE
API-11210: BUG/Add default value to notice acknowledgement

### PR DESCRIPTION
**Pull request by ElleeB**
Bug ticket: https://vajira.max.gov/browse/API-11210

This was a miss. We'll always check 'YES' for notice acknowledgement since those with a falsey value will have already been filtered out.

### CODE REVIEW

**Review by nathanbwright — COMMENTED** [review]
It doesn't feel right to me to make this always true in the model, because it's a required field and the model isn't valid unless it's true. I know that we are checking at the controller level, but I'm still uneasy about doing it here. Shouldn't we be doing this at the seed data level (for our specs)?

**Comment by nathanbwright** [comment]
I went back and looked at the seed data (https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/fixtures/valid_200995.json#L21) and it just looks like a name mismatch `data_attributes` vs `dataAttributes`!

**Comment by ElleeB** [comment]
> It doesn't feel right to me to make this always true in the model, because it's a required field and the model isn't valid unless it's true. I know that we are checking at the controller level, but I'm still uneasy about doing it here. Shouldn't we be doing this at the seed data level (for our specs)?

Yeah I went back and forth. I'll go ahead and make it feel better!

**Comment by ElleeB** [comment]
> I went back and looked at the seed data (https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/fixtures/valid_200995.json#L21) and it just looks like a name mismatch `data_attributes` vs `dataAttributes`!

I'll take a look! Thank you :)

File: modules/appeals_api/app/models/appeals_api/supplemental_claim.rb [hunk 1]
```diff
@@ -176,7 +176,7 @@ def new_evidence_dates
     end
 
     def notice_acknowledgement
-      data_attributes['notice_acknowledgement']
```
Comment (nathanbwright) [inline modules/appeals_api/app/models/appeals_api/supplemental_claim.rb:179]: I'm probably missing something here... Is there a reason we need `? 1 : 'Off'` here? Can't it just be true/false?

File: modules/appeals_api/app/models/appeals_api/supplemental_claim.rb [hunk 2]
```diff
@@ -176,7 +176,7 @@ def new_evidence_dates
...
+      data_attributes['noticeAcknowledgement'] ? 1 : 'Off'
```
Comment (nathanbwright) [inline modules/appeals_api/app/models/appeals_api/supplemental_claim.rb:179]: I'm probably missing something here... Is there a reason we need `? 1 : 'Off'` here? Can't it just be true/false?

Comment (ElleeB) [reply modules/appeals_api/app/models/appeals_api/supplemental_claim.rb:179]: The value consumed by the structure needs to be 1 or 'Off' if I'm not mistaken.

**Review by cilestin — DISMISSED** [review]

**Review by cilestin — APPROVED** [review]

**Pull request merged**